client/src/index.jsx

```diff
@@ -251,8 +282,10 @@ export default class App extends React.Component {
           <div className="left auto-center">
             <button onClick={() => this.handlePhotoNavigationClick(-1)}><i className="fa fa-5x fa-angle-left left-middle" aria-hidden="true" ></i></button>
           </div>
-          <Carousel currentPhoto={this.src[this.state.currentPhotoIndex]} />
+          <Carousel registerFavorite={this.registerFavorite.bind(this)} currentPhoto={this.src[this.state.currentPhotoIndex]} />
```

Comment:
Nitpick not worth fixing, but I would put registerFavorite second since it's not the first method that will be called when the carousel component loads.